### PR TITLE
[ZT] Azure risky users

### PR DESCRIPTION
**Pull request by ranbel**
#7121

### CODE REVIEW

**Comment by ranbel** [comment]
https://ranbel-azure-risky-users.cloudflare-docs-7ou.pages.dev/cloudflare-one/tutorials/azuread-risky-users/

File: content/cloudflare-one/tutorials/azuread-risky-users.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+1. Authenticate `wrangler`.
+
+    ```sh
+    $ wrangler login
+    ```
+
+2. Open a terminal and clone our example project.
+
+    ```sh
+    $ wrangler generate risky-users https://github.com/cloudflare/<repo-name>
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/azuread-risky-users.md]: Worker needs to be added to the Cloudflare repo.

File: content/cloudflare-one/tutorials/azuread-risky-users.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+## 5. Create a browser isolation policy
+
+Finally, create a [Gateway HTTP policy](/cloudflare-one/policies/filtering/http-policies/) to isolate traffic for risky user groups.
+
+1. In the [Zero Trust dashboard](https://dash.teams.cloudflare.com), go to **Gateway** > **Policies** > **HTTP**.
+
+2. Select **Create a policy**.
+
+3. Build an [Isolate policy](/cloudflare-one/policies/browser-isolation/isolation-policies/) that contains a _User Group Names_ rule. For example, the following policy serves `app1.example.com` and `app2.example.com` in a remote browser for all members flagged as high risk:
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/azuread-risky-users.md:160]: Will the SCIM groups automatically appear in the Gateway policy builder?

File: content/cloudflare-one/tutorials/azuread-risky-users.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/tutorials/azuread-risky-users.md]:
```suggestion
    $ wrangler generate risky-users https://github.com/cloudflare/msft-risky-user-ad-sync
```

**Review by obezuk — APPROVED** [review]

**Pull request merged**